packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1871,6 +1893,10 @@ export default class TradeStore extends BaseStore {
         return isVanillaFxContract(this.contract_type, this.symbol);
     }
 
+    get is_touch() {
```

Comment:
Dow we need to add all new stuff to mockStore?

Reply:
yes, let me add it

Reply:
@kate-deriv I've added it to the mock store. Please review again.